tools/clang/test/HLSLFileCheck/validation/CallableParamIsStruct.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: %dxv %s | FileCheck %s
```

Comment:
Could we add validator version in the %dxv command line in case validation rule changed in the future?

Reply:
I'd prefer not to because the rule is valid on future versions until we change that. I'd rather we add a version when we change it so that we get a failure on changing the rules.

Reply:
We can require a minimum version with `%dxilver`, but not a maximum.  We should be using `%dxilver` for validation rules added/changed at some version.  In ValidationTest.cpp, we use something like: `if (m_ver.SkipDxilVersion(1, 3)) return;` to enforce a minimum of 1.3.  For tests ported from there, we should retain this through the use of `%dxilver`.

Instead of having a maximum version, we should probably just remove the test, since it's only testing something we no longer want.

When adding/updating validation tests, it would be best to verify that they pass (and skip appropriately) when run against that last few external validators we released.